content/en/about/contributing/syntax/_index.md

```diff
@@ -22,6 +22,42 @@ For references The following shortcodes are available:
 - [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/)
 - [Docsy theme shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/)
 
+### Adding chapters
```

Comment:
I thought you removed/renamed the parts about chapters?
Please replace by section for now.

Reply:
```suggestion
### Adding sections
```

Reply:
I did, but then stashed content-related changes while on call with you.